### ISSUE TITLE
TypeScript support breaks Angular

### ISSUE THREAD

**Issue by otto-dev** [otto-dev/coordinate-parser#5]
_Originally posted by @jgeiger-cr in https://github.com/otto-dev/coordinate-parser/pull/4#issuecomment-696301351_

The TypeScript types seems to break the Angular use case. 

```
import * as Coordinates from 'coordinate-parser';
export class Location {
  static tryParse(candidate: string): Location {
    try {
       const candidateAsLocation: any = new Coordinates(candidate);
       return new Location(candidateAsLocation.getLatitude(), candidateAsLocation.getLongitude());
     } catch {
       return null;
     }
   }
}

```
`error TS2351: This expression is not constructable.
Type 'typeof import("coordinate-parser")' has no construct signatures.
38       const candidateAsLocation: any = new Coordinates(candidate);
`

**Comment by otto-dev** [comment]
@jgeiger-cr What version of TypeScript are you using?

**Comment by otto-dev** [comment]
Should be fixed in #6 . I'm not a TypeScript user, please confirm.

**Comment by otto-dev** [comment]
Confirmed fixed. Thanks!
